### PR TITLE
feat: add project pipeline process and update style

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
feat: add project pipeline process and update style

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb24iOls4ODMsNzcyXSwibWVtYmVyIjpbIjEwMDEyNjEiXX0%3D&id=275573&iterationID=772&pId=0&type=REQUIREMENT)

#### Specified Reviewers:

/assign @sfwn @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     feat: add project pipeline process and update style         |
| 🇨🇳 中文    |      项目流水线增加进度、修改表样式       |

#### Need cherry-pick to release versions?
/cherry-pick release/1.6-alpha.2
Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/pipelineTable/provider.go [hunk 1]
```diff
@@ -411,6 +417,17 @@ func (p *PipelineTable) SetTableMoreOpItem(definition *pb.PipelineDefinition, de
 				}
 				return "cron"
 			}()),
+
+			Icon: func() *commodel.Icon {
+				if *v.Enable {
+					return &commodel.Icon{
+						Type: "shijian",
```
Comment (liuhaoyang) [inline modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/pipelineTable/provider.go:424]: Irregular naming

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/pipelineTable/provider.go (hunk 1 above)
Comment (littlejiancc) [reply modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/pipelineTable/provider.go:430]: done

**Review by kakj-go — APPROVED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**